source/solvers/gls_navier_stokes.cc

```diff
@@ -445,9 +450,9 @@ GLSNavierStokesSolver<dim>::assembleGLS()
               const double JxW = fe_values.JxW(q);
 
               // Calculation of the GLS stabilization parameter. The
-              // stabilization parameter used is different if the simulation is
-              // steady or unsteady. In the unsteady case it includes the value
-              // of the time-step
+              // stabilization parameter used is different if the simulation
+              // is steady or unsteady. In the unsteady case it includes the
+              // value of the time-step
```

Comment:
These are weird changes? Were they caused by the automatic indentation?

Reply:
certainly, I do not recall having chanced these!